app/exiv2app.hpp

```diff
@@ -12,49 +12,61 @@
 // included header files
 #include <exiv2/exiv2.hpp>
 
-#include "types.hpp"
 #include "getopt.hpp"
+#include "types.hpp"
 
 // + standard includes
-#include <set>
 #include <iostream>
 #include <regex>
+#include <set>
 
 // *****************************************************************************
 // class definitions
 
 //! Command identifiers
-enum CmdId { invalidCmdId, add, set, del, reg };
+enum CmdId
+{
+    invalidCmdId,
+    add,
+    set,
+    del,
+    reg
+};
 //! Metadata identifiers
 // enum MetadataId { invalidMetadataId, iptc, exif, xmp };
 //! Metadata identifiers
 // mdNone=0, mdExif=1, mdIptc=2, mdComment=4, mdXmp=8
-enum MetadataId { invalidMetadataId = Exiv2::mdNone
-                , iptc = Exiv2::mdIptc
-                , exif = Exiv2::mdExif
-                , xmp  = Exiv2::mdXmp
-                } ;
+enum MetadataId
+{
+    invalidMetadataId = Exiv2::mdNone,
+    iptc = Exiv2::mdIptc,
+    exif = Exiv2::mdExif,
+    xmp = Exiv2::mdXmp
+};
 
 //! Structure for one parsed modification command
-struct ModifyCmd {
+struct ModifyCmd
+{
     //! C'tor
-    ModifyCmd() :
-        cmdId_(invalidCmdId), metadataId_(invalidMetadataId),
-        typeId_(Exiv2::invalidTypeId), explicitType_(false) {}
-    CmdId cmdId_;                               //!< Command identifier
-    std::string key_;                           //!< Exiv2 key string
-    MetadataId metadataId_;                     //!< Metadata identifier
-    Exiv2::TypeId typeId_;                      //!< Exiv2 type identifier
+    ModifyCmd()
+        : cmdId_(invalidCmdId), metadataId_(invalidMetadataId), typeId_(Exiv2::invalidTypeId), explicitType_(false)
+    {
+    }
+    CmdId cmdId_;            //!< Command identifier
+    std::string key_;        //!< Exiv2 key string
+    MetadataId metadataId_;  //!< Metadata identifier
+    Exiv2::TypeId typeId_;   //!< Exiv2 type identifier
     //! Flag to indicate if the type was explicitly specified (true)
     bool explicitType_;
-    std::string value_;                         //!< Data
+    std::string value_;  //!< Data
 };
 //! Container for modification commands
 using ModifyCmds = std::vector<ModifyCmd>;
 //! Structure to link command identifiers to strings
-struct CmdIdAndString {
-    CmdId cmdId_;                               //!< Commands identifier
-    std::string cmdString_;                     //!< Command string
+struct CmdIdAndString
```

Comment:
Feels like a lot of these can be converted to std::pair + structured bindings.

Reply:
Please, let's focus in this PR in the clang-format style. I am sure you will find many opportunities here for refactoring but these should be addressed in other PRs :wink: